draft-amjad-cfrg-partially-blind-rsa.md

```diff
@@ -383,63 +382,64 @@ In contrast, if the PrepareRandomize function is used, then the application
 message is `slice(input_msg, 32, len(input_msg))`, i.e., the prepared message
 with the random prefix removed.
 
-## Public Key Augmentation {#augment-public-key}
+## Public Key Derivation {#augment-public-key}
 
-The public key augmentation function (AugmentPublicKey) derives a per-metadata public
+The public key derivation function (DerivePublicKey) derives a per-metadata public
 key that is used in the core protocol. The hash function used for HKDF is that which
 is associated with the RSAPBSSA instance and denoted by the `Hash` parameter. Note that
 the input to HKDF is expanded to account for bias in the output distribution.
 
 ~~~
-AugmentPublicKey(pkS, metadata)
+DerivePublicKey(info)
```

Comment:
This will take n aka modulus as input. Some calls are passing pk to it and some aren't. All calls need to be corrected to pass modulus to DerivePublicKey.

Reply:
Ah, yes, good catch! I'll add `pk` back to the list of parameters, since that's what has `n`.